ClassTranscribeDatabase/Globals.cs

```diff
@@ -17,6 +18,9 @@ public class AppSettings
         public string ADMIN_USER_ID { get; set; }
         public string ADMIN_PASSWORD { get; set; }
         public string RabbitMQServer { get; set; }
+
```

Comment:
we should be consistent with variables like this. I think it be good to look at a linter that checks formatting (yes I care about that).

See 
- https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions
- https://google.github.io/styleguide/csharp-style.html

Might be interesting to look at:
https://marketplace.visualstudio.com/items?itemName=ChrisDahlberg.StyleCop